Dshop/apps/products_catalogue/models.py

```diff
@@ -28,13 +29,18 @@ class Meta:
         verbose_name_plural = 'Categories'
 
 
-class Product(models.Model):
-    name = models.CharField(max_length=255)
+class Product(CatalogueItemModel):
     category = models.ForeignKey(Category, on_delete=models.CASCADE)
     price = models.DecimalField(max_digits=10, decimal_places=2)
     price_last_30_days = models.DecimalField(max_digits=10, decimal_places=2, null=True, blank=True)
     short_description = models.TextField()
     full_description = models.TextField()
+    created_at = models.DateTimeField(default=timezone.now, editable=False)
+    updated_at = models.DateTimeField(default=timezone.now)
```

Comment:
Please change to the below and create migration, of course. 
```
    created_at = models.DateTimeField(auto_now_add=True)
    updated_at = models.DateTimeField(auto_now=True)
```
The code you provided won't work - it will only save date of product creation in db. The field I added are updated automatically by Django upon object update and create.